paystell-frontend/src/app/layout.tsx

```
@@ -29,6 +44,36 @@ export default function RootLayout({
             </AuthProvider>
           </WalletProvider>
         </ThemeProvider>
+        <script
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)